astropy/cosmology/flrw/base.py

```diff
@@ -66,7 +66,32 @@ def quad(*args, **kwargs):
 ##############################################################################
 
 
-class FLRW(Cosmology):
+class _ScaleFactorMixin:
+    @property
+    def scale_factor0(self):
+        """Scale factor at redshift 0."""
+        return u.Quantity(1.0, unit=u.one)
```

Comment:
I think this would be better as `return self.scale_factor(0)` (or `self.scale_factor` should use `self.scale_factor0`). There would seem to be no need for speed, but defensive programming against divergence is never bad...

Reply:
``return self.scale_factor(0)`` sounds good, so long as it's identically 1!